diff --git a/setup.py b/setup.py
index cb304ba..69aafdd 100644
--- a/setup.py
+++ b/setup.py
@@ -29,7 +29,7 @@ from setuptools.command import build_ext
 from setuptools.command import build_py
 from setuptools.command import install
 
-__version__ = 'dev'
+__version__ = '0.8.11'
 IS_WINDOWS = (platform.system() == 'Windows')
 MP_ROOT_PATH = os.path.dirname(os.path.abspath(__file__))
 MP_DIR_INIT_PY = os.path.join(MP_ROOT_PATH, 'mediapipe/__init__.py')
@@ -366,6 +366,8 @@ class BuildPy(build_py.build_py):
     build_py.build_py.finalize_options(self)
 
   def run(self):
+    self.link_opencv = True
+
     _modify_opencv_cmake_rule(self.link_opencv)
     _add_mp_init_files()
     build_modules_obj = self.distribution.get_command_obj('build_modules')
@@ -424,9 +426,9 @@ class Restore(setuptools.Command):
 
 
 setuptools.setup(
-    name='mediapipe',
+    name='mediapipe-silicon',
     version=__version__,
-    url='https://github.com/google/mediapipe',
+    url='https://github.com/cansik/mediapipe-silicon',
     description='MediaPipe is the simplest way for researchers and developers to build world-class ML solutions and applications for mobile, edge, cloud and the web.',
     author='The MediaPipe Authors',
     author_email='mediapipe@google.com',
